## <span style="color:darkblue"> Midterm Exam

Stat140-02: Introduction to the Ideas and Applications of Statistics 

Due: Wednesday, March 7 @11am

*Under no circumstancs should a student discuss the content of this exam with other students or the TAs. All students are expected to follow the Honor Code: “I will honor myself, my fellow students, and Mount Holyoke College by acting responsibly, honestly, and respectfully in both my words and deeds.” Students are encouraged to speak with the instructor if unclear about what constitutes an Honor Code violation. Violations of the Honor Code may result in failure of this course.* 

**Problem 1 (40 points)**

The human gastrointestinal system is home to a vast number of microbial cells; the term "human microbiome" refers specifically to the full collection of genes of all microbes in this community. Gut microbial communities are known to influence host health or disease susceptibility.

In this problem, you will examine data adapted from a study investigating the role of the microbiome in the development of colon cancer (Kostic, Aleksandar D., et al., 2012, Genomic analysis identifies association of *Fusobacterium* with colorectal carcinoma. Genome Research 22: 292-298.) The data are read into `R` using the code below.

The original study examined cancer tissue and adjacent normal tissue collected from colon cancer patients living in Spain, Vietnam, and the United States. Recorded measurements include information about the patient (e.g. age, gender) and about the microbes found in the colon. When the bacteria are measured in this setting, the data for each species (i.e., for each "bug") are recorded as the frequency of that species detected divided by the total number of bacteria from all species. Thus, the values represent the proportion of bacteria present belonging to that particular species; this type of data is often referred to as relative abundance data, or abundance data. 

*Among these 100 bacteria, which seem to have unexpectedly large or small levels of mean relative abundance? In other words, which bacteria have a mean relative abundance that is an outlier?*

The data you will work with includes abundance data for only the cancer tissue samples from each patient. The dataset pheno.abundance includes the following information for each patient:

  - `Age`: age at diagnosis, in years
  - `Gender`: gender, coded as Male or Female
  - `Stage`: coded 1-4.  Larger values of Stage indicate cancer that is harder to treat
  - `bug1` to `bug100`: relative abundance for each bacteria (i.e., "bug") identified in the tumor samples. The numbers represent the proportion of the total number of bacteria per sample that were identified as a particular type; for example, a value of 0.05 in `bug1` for a patient indicates that 5% of the gut bacteria in that patient were of type `bug1`.(DNA sequencing was used to identify the species of each bug type, but for simplicity, we have not included that information.) 

In [12]:
load("pheno_abundance.Rdata")
head(pheno.abundance)

Age Gender Stage bug1         bug2       bug3         bug4      
1  71  Female 2     0.0255190311 0.09212803 0.5240051903 0.00000000
3  53  Female 4     0.1600400936 0.08419646 0.0755095222 0.00000000
5  55  Female 2     0.0008966599 0.01053575 0.3120376597 0.00000000
7  44  Male   2     0.1072625698 0.13798883 0.0005586592 0.07150838
9  83  Female 1     0.2434544744 0.08010942 0.0000000000 0.02461899
11 52  Male   3     0.0015105740 0.02658610 0.2000000000 0.04894260
   bug5         bug6 bug7      ⋯ bug91       bug92 bug93      bug94       
1  0.0000000000 0    0.0000000 ⋯ 0.000216263 0     0.00000000 0.0000000000
3  0.0006682259 0    0.0000000 ⋯ 0.022385566 0     0.00000000 0.0006682259
5  0.0000000000 0    0.0000000 ⋯ 0.000224165 0     0.00000000 0.0000000000
7  0.0525139665 0    0.1307263 ⋯ 0.000000000 0     0.02122905 0.0000000000
9  0.0027354435 0    0.0000000 ⋯ 0.000000000 0     0.00000000 0.0000000000
11 0.0036253776 0    0.1456193 ⋯ 0.026283988 0     0.00000000 0.0006042296
   bug95        bug96      bug97        bug98       bug99       bug100   
1  0.0000000000 0.00000000 0.0000000000 0.000000000 0.000000000 0.0000000
3  0.0000000000 0.00000000 0.0000000000 0.000000000 0.000000000 0.0000000
5  0.0000000000 0.01770903 0.0000000000 0.000000000 0.000000000 0.1259807
7  0.0000000000 0.00000000 0.0067039106 0.000000000 0.006703911 0.0000000
9  0.0000000000 0.00000000 0.0003907776 0.002344666 0.000000000 0.0000000
11 0.0006042296 0.00000000 0.0000000000 0.000000000 0.003021148 0.0000000

a) We can calculate the median abundance for each bug across all patients using the following `R` code:   

In [2]:
library(dplyr)
bugMedians = pheno.abundance[,4:103] %>% summarise_all(funs(median))
bugMedians


Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union



bug1       bug2       bug3      bug4       bug5        bug6 bug7 bug8 bug9
1 0.02621518 0.06978119 0.0646438 0.02012651 0.006583278 0    0    0    0   
  bug10       ⋯ bug91       bug92 bug93 bug94 bug95 bug96 bug97 bug98       
1 0.005438066 ⋯ 0.000216263 0     0     0     0     0     0     0.0001884659
  bug99 bug100
1 0     0

To create a new variable in a data frame, we use the following code:

In [3]:
df = data.frame(medianAbundance = as.numeric(bugMedians))
head(df)

medianAbundance
1 0.026215183    
2 0.069781195    
3 0.064643799    
4 0.020126509    
5 0.006583278    
6 0.000000000

Create a new variable called `meanAbundance` that is equal to the **mean** abundance for each bug across all patients and explain in terms of the data what the values of this variable represent. 

b) Using both numerical and graphical summaries, describe the distribution of the mean abundance levels across all bugs.

c)  Identify the outliers in mean abundance level. Which outliers have mean relative abundances that qualify as outliers? 

d) Now considering just bug 3, use graphical and numerical summaries to describe the distribution of abundance levels across all patients for each stage of cancer. What can you conclude based on these summaries?

In [16]:
pheno.abundance%>%
filter(Stage==1) %>%
summarize(mean=mean(bug3),
          median=median(bug3),
          min=min(bug3),
          max=max(bug3),
          q1=quantile(bug3, p=.25),
          q3=quantile(bug3, p=.75),
          IQR=q3-q1,
          upper_whisker=q3+1.5*IQR,
          lower_whisker=q1-1.5*IQR)


mean      median    min max       q1          q3       IQR      
1 0.1976714 0.2577713 0   0.3672566 0.005323194 0.358006 0.3526828
  upper_whisker lower_whisker
1 0.8870303     -0.5237011

#### Problem 2 (30 points)

In Eastern Europe there are epidemics of both tuberculosis and HIV/AIDS. There is some research which suggests that these diseases may not occur independently. Suppose that the table below shows the probability that a randomly chosen citizen of an Eastern European country has the disease status indicated in the table.

| |  **HIV Status ** | |
|--|--|--|--|
| **Tuberculosis** | Positive (HIV$^+$) | Negative (HIV$^-$)| 
| Present |  0.15 |   0.15  | 
| Absent |  0.10 |  0.60  | 

	

a) What is the prevalence of Tuberculosis (TB)?

b) Suppose $A$ is the event that a randomly chosen member of the population is HIV$^+$ and $B$ is the event that a randomly chosen person has TB. Are $A$ and $B$ independent? Justify your answer.

c) Based on the definitions of $A$ and $B$ in part (b), what is the conditional probability $P(A|B)$?

d) Among HIV$^+$ individuals, what is the probability that someone has TB?

#### Problem 3 (15 points)

Suppose a 60-year-old man who has never smoked cigarettes presents to a physician with symptoms of a chronic cough and occasional breathlessness. The physician becomes concerned and orders the patient admitted to the hospital for a lung biopsy. Suppose the results of the biopsy are consistent either with lung cancer or sarcoidosis (a fairly common, nonfatal lung disease).

Let $A = \{\text{chronic cough, results of lung biopsy}\}$. Partition the patient's possible disease state into: $B_1 = \text{normal}, B_2 = \text{lung cancer}, B_3 = \text{sarcoidosis}$. 

From clinical experience, the physician estimates $P(A|B_i)$: $P(A|B_1) = .001$, $P(A|B_2) = .9$, and $P(A|B_3) = .9$. Suppose that in 60-year-old, never-smoking men $P(B_1) = .99$, $P(B_2) = .001$, and $P(B_3) = .009$. 

Find the probabilities of the three disease states given the previous symptoms: $P(B_1|A)$, $P(B_2|A)$, and $P(B_3|A)$. Which is the most likely?


#### Problem 4 (15 points)

High blood cholesterol levels increase the risk of heart disease. Young women are generally less afflicted with high
cholesterol than other groups. The cholesterol levels for women aged 20 to 34 follow an approximately normal distribution with mean 185 milligrams per deciliter (mg/dl) and standard deviation 39 mg/dl.

a) Cholesterol levels above 240 mg/dl demand medical attention. What percent of young women have levels above 240 mg/dl?

b) Levels above 200 mg/dl are considered borderline high. What percent of young women have blood cholesterol between 200 and 240 mg/dl?